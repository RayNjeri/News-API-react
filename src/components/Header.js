import React, { Component } from 'react';
import ReactDOM from 'react-dom';

export default class Header extends Component {
    render() {
        return (
            <div>
                <h1>THE NEWS HUB.</h1>
            </div>
        );
    }
}

