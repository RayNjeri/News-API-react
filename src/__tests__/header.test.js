import React from 'react';
import { mount, shallow } from 'enzyme';
import shallowToJSON from 'enzyme-to-json';
import Header from '../components/Header.js';

describe('Header Component', () => {
    it('Snapshot of Header Component', () => {
        const headers = shallow(<Header />);
        // console.log(headers.debug());
        const tree = shallowToJSON(headers);
        expect(headers.containsMatchingElement(<h1>THE NEWS HUB.</h1>)).toBe(true);
        // expect(headers.containsMatchingElement(<h1 />)).toBe(true)
        expect(tree).toMatchSnapshot();
    });
});
