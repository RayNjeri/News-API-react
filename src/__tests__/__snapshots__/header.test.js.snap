// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Snapshot of Header Component 1`] = `
<div>
  <h1>
    THE NEWS HUB.
  </h1>
</div>
`;
